README.md

```diff
@@ -78,14 +78,22 @@ those that could be used for actionable alerts. Please contribute PR's for
 additional metrics!
 
 > WARNING: THESE METRIC/TAG NAMES ARE UNSTABLE AND MAY CHANGE IN A FUTURE RELEASE.
-> For now the following metrics
+> For now the following metrics and collectors
 >
+> **metrics**
 >	* kube_pod_container_resource_requests_nvidia_gpu_devices
 >	* kube_pod_container_resource_limits_nvidia_gpu_devices
 >	* kube_node_status_capacity_nvidia_gpu_cards
 >	* kube_node_status_allocatable_nvidia_gpu_cards
 >
 >	are in alpha stage and will be deprecated when the kubernetes gpu support is final in 1.9 version.
+>
+> **collectors**
```

Comment:
In addition I think we can add a catch all statement for all collectors and metrics based on alpha features, we cannot provide any stability guarantees.

Reply:
would we split `collector and metrics` statement info into each collector README or create a new file?

Reply:
Just a statement above here that says something like: "Any collectors and metrics based on alpha Kubernetes APIs are excluded from *any* stability guarantee. They may change at any given release."

Reply:
got it, thank you @brancz